[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:00:12,167 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:00:25,580 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:00:58,285 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:01:15,172 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:01:30,947 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:02:03,558 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:02:18,173 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:02:36,489 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:03:09,415 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:03:21,159 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:03:42,228 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:04:15,039 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:04:24,170 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:04:47,714 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:05:20,611 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:05:27,162 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:05:53,613 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:06:26,381 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:06:30,197 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:06:59,007 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:07:31,759 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:07:33,173 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:08:04,422 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:08:36,164 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:08:37,066 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:09:09,825 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:09:39,164 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:09:42,471 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:10:15,141 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:10:42,169 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:10:47,826 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:11:20,529 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:11:45,169 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:11:53,476 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:12:26,451 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:12:48,170 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:12:59,402 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:13:32,380 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:13:51,178 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:14:05,277 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:14:38,176 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:14:54,226 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:15:10,917 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:15:43,589 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:15:57,166 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:16:16,371 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:16:49,086 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:17:00,183 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:17:21,757 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:17:54,735 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:18:03,168 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:18:27,615 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:19:00,281 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:19:06,162 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:19:32,994 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:20:05,761 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:20:09,175 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:20:38,734 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:21:11,777 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:21:12,179 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:21:44,650 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:22:15,172 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:22:17,456 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:22:50,260 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:23:18,166 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:23:22,990 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:23:55,638 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:24:21,166 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:24:28,334 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:25:01,211 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:25:24,157 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:25:33,995 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:26:07,016 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:26:27,166 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:26:40,050 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:27:12,686 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:27:30,202 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:27:45,546 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:28:18,535 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:28:33,168 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:28:51,263 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:29:24,074 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:29:36,172 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:29:56,745 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:30:29,545 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:30:39,181 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:31:02,476 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:31:35,403 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:31:42,159 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:32:08,314 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:32:41,163 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:32:45,166 ERROR [dubbo-client-idleCheck-thread-1][ReconnectTimerTask.java:51] -  [DUBBO] Fail to connect to HeaderExchangeClient [channel=org.apache.dubbo.remoting.transport.netty4.NettyClient [192.168.56.1:0 -> /192.168.56.1:20881]], dubbo version: 2.7.3, current host: 192.168.56.1
org.apache.dubbo.remoting.RemotingException: client(url: dubbo://192.168.56.1:20881/com.alibaba.cloud.dubbo.service.DubboMetadataService?anyhost=true&application=user-service&bind.ip=192.168.56.1&bind.port=20881&check=false&codec=dubbo&deprecated=false&dubbo=2.0.2&dynamic=true&generic=true&group=uaa-service&heartbeat=60000&interface=com.alibaba.cloud.dubbo.service.DubboMetadataService&lazy=false&methods=getAllServiceKeys,getServiceRestMetadata,getExportedURLs,getAllExportedURLs&pid=50556&qos.enable=false&qos.port=22230&register=true&register.ip=192.168.56.1&release=2.7.3&remote.application=uaa-service&retries=-1&revision=2.1.0.RELEASE&side=consumer&sticky=false&timeout=3000&timestamp=1712717623321&version=1.0.0) failed to connect to server /192.168.56.1:20881, error message is:Connection refused: no further information: /192.168.56.1:20881
	at org.apache.dubbo.remoting.transport.netty4.NettyClient.doConnect(NettyClient.java:166)
	at org.apache.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:190)
	at org.apache.dubbo.remoting.transport.AbstractClient.reconnect(AbstractClient.java:246)
	at org.apache.dubbo.remoting.exchange.support.header.HeaderExchangeClient.reconnect(HeaderExchangeClient.java:155)
	at org.apache.dubbo.remoting.exchange.support.header.ReconnectTimerTask.doTask(ReconnectTimerTask.java:49)
	at org.apache.dubbo.remoting.exchange.support.header.AbstractTimerTask.run(AbstractTimerTask.java:87)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:648)
	at org.apache.dubbo.common.timer.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:727)
	at org.apache.dubbo.common.timer.HashedWheelTimer$Worker.run(HashedWheelTimer.java:449)
	at java.lang.Thread.run(Thread.java:745)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: /192.168.56.1:20881
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:327)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:665)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:612)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:529)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:491)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:905)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	... 1 more
Caused by: java.net.ConnectException: Connection refused: no further information
	... 11 more
[${project.name}][${env:SERVER_PORT}] 2024-04-11 21:33:13,853 ERROR [com.alibaba.nacos.client.Worker.longPolling.fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f][CacheData.java:198] - [fixed-127.0.0.1_8848-416dd4fd-f3e7-4059-b48f-58bbe4be965f] [notify-error] dataId=user-service.yaml, group=SHANJUPAY_GROUP, md5=b696bf28e976fa8ec4b40ab863af078d, listener=com.alibaba.cloud.nacos.refresh.NacosContextRefresher$1@6d9ebe8a tx=org.springframework.boot.context.properties.bind.BindException: Failed to bind properties under 'spring.datasource.druid' to javax.sql.DataSource
